<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLiveWidgetDemoClass</class>
 <widget class="QWidget" name="QtLiveWidgetDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtLiveWidgetDemo</string>
  </property>
  <widget class="QLabel" name="VideoLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>933</width>
     <height>579</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>630</y>
     <width>207</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="OpenLive">
      <property name="text">
       <string>OpenCamera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RecordScreen">
      <property name="text">
       <string>RecordScreen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtLiveWidgetDemo.qrc"/>
 </resources>
 <connections/>
</ui>
